shell/weights/update_training_history.py

```diff
@@ -75,13 +75,33 @@
 tensorboard_results = tensorboard_experiment.get_scalars()
 
 training_epochs = max(tensorboard_results[tensorboard_results.run == "train"].step)
-max_validation_accuracy = max(
-    tensorboard_results[
-        (tensorboard_results.run == "validation")
-        & (tensorboard_results.tag == "epoch_categorical_accuracy")
-    ].value
-)
-max_validation_accuracy = f"{max_validation_accuracy:.4f}"
+
+results_tags = tensorboard_results.tag.unique()
```

Comment:
maybe comment the code?

Reply:
Good call -- added some comments about the val accuracy and mean iou